metaworld/envs/mujoco/multitask_env.py

```diff
@@ -100,6 +101,10 @@ def __init__(self,
         self._sample_goals = sample_goals
         self._obs_type = obs_type
 
+        self._max_obs_dim = max([
+            np.prod(env_cls().observation_space.shape) for env_cls in task_env_cls_dict.values()
```

Comment:
for MT50, this will still consume as much memory as all 50 envs (because it constructs all of them)

is this what you intended?

Reply:
Not what I intended, but I can't think of another way to find the maximum observation dimension, except we put that information in the dict like `MEDIUM_MODE_ARGS_KWARGS`